AddOns/BasicFinancials/app/src/page/AssistedSetup.Page.al

```diff
@@ -0,0 +1,131 @@
+page 20600 "Assisted Setup BF"
+{
+    Caption = 'Basic Financials Assisted Setup';
+    DeleteAllowed = false;
+    InsertAllowed = false;
+    LinksAllowed = false;
+    PageType = NavigatePage;
+    ShowFilter = false;
+
+    layout
+    {
+        area(content)
+        {
+            label(Title)
+            {
+                Caption = 'Welcome';
+                Style = StandardAccent;
+                ApplicationArea = All;
+            }
+            group(Instructional)
+            {
+                Caption = ' ';
+                InstructionalText = 'The Basic Financials Extensions aligns the User Experience to the license limitations.';
```

Comment:
The Basic Financials extension enables the subset Dynamics 365 Business Central capabilities provided by the Basic Financials license.